### ISSUE TITLE
Interaction Bits from mod: Chisels and Bits

### ISSUE THREAD

**Issue by PletPeter** [VsnGamer/ElevatorMod#71]
Elevators can get the appearance of blocks by right clicking with a block in hand. However this doesn't work with bits and leads to what is essentially a chunkban.

**To Reproduce**

Put an elevator block in the world
Rightclick the elevator with a bit from chisels and bits in hand (possibly same with bigger pieces from the mod)
Your client will kick you and unless you are still in the area upon relogging it will keep kicking you.

**Expected behavior**
Either get the appearance of the bit or have nothing happen.

**Comment by VsnGamer** [comment]
Can't reproduce, maybe it only occurs with non-vanilla blocks bits?
Also what version are you using?
![image](https://user-images.githubusercontent.com/18006201/103686541-e0c2b200-4f86-11eb-9d2f-8e3e5f51c32c.png)

**Comment by PletPeter** [comment]
Did a little bit of testing myself, I erred by saying it was a single bit.

I followed the following steps in single-player to reproduce it:

1. Placed down a glowstone block
2. Changed the mode of the Chisel by shift-rightclicking with the chisel in hand till it selects the mode that takes layers off the block
3. Took layers off till I only have a single layered block left (have a bit-bag in your inventory to save the bits so you don't have to keep throwing them away).
4. broke the last  layer with my fists and picked it up.
5. right-clicked the elevator with it. Crash ensues.

Crash log to launcher in the [pastebin](https://pastebin.com/kHNrJdKa)
